apps/rush-lib/src/api/RushConfiguration.ts

```diff
@@ -1391,6 +1418,14 @@ export class RushConfiguration {
     return this._telemetryEnabled;
   }
 
+  public get projectsExtended(): string[] {
+    return this._projectsExtended || [];
+  }
+
+  public set projectsExtended(vals: string[]) {
```

Comment:
We aren't usually in the habit of making the configuration mutable through this object, since it doesn't include logic to write changes.